site/content/en/docs/Guides/fleet-updates.md

```diff
@@ -26,12 +26,21 @@ much of the `Fleet` is  incremented and decremented at one time:
       maxUnavailable: 25%
 ```
 
-So when a Fleet is edited, either through `kubectl` `edit`/`apply` or via the Kubernetes API, this performs the following operations:
+So when a Fleet is edited (any field other than `replicas`, see note below), either through `kubectl` `edit`/`apply` or via the Kubernetes API, this performs the following operations:
 
 1. Adds the `maxSurge` number of `GameServers` to the Fleet.
 1. Shutdown the `maxUnavailable` number of `GameServers` in the Fleet, skipping `Allocated` `GameServers`.
 1. Repeat above steps until all the previous `GameServer` configurations have been `Shutdown` and deleted.
 
+{{< alert title="Note" color="info">}}
+When fleet update contains changes of `replicas` parameter, than new gameservers would be created/deleted straight away,
```

Comment:
Just making a note from a conversation in chat - looks like this isn't 100% accurate, since you can get rolling updates even with a `replica` change.

Reply:
yes, if there is only `replicas` change then there would be fast scale up, say from 3 to 30. If `replicas` + `label` then it would go stepwise.

Reply:
Gentle bump on this - should we be adding some changes to the above to document the chat conversation we had?

Reply:
Yes, definetely. Editing this